README.md

```diff
@@ -22,7 +22,6 @@ applications running in almost any environment. Here's an introductory video:
 1. Your application needs to use Node.js version 0.12 or greater.
 1. You need a [Google Cloud project](https://console.cloud.google.com). Your application can run anywhere, but errors are reported to a particular project.
 1. [Enable the Stackdriver Error Reporting API](https://console.cloud.google.com/apis/api/clouderrorreporting.googleapis.com/overview) for your project.
-1. The module will only send errors when the `NODE_ENV` environment variable is set to `production`.
```

Comment:
I was going to suggest that this be modified to mention that we either need `NODE_ENV` or the `ignoreEnvironmentCheck` configuration to send errors but it looks like `ignoreEnvironmentCheck` is true by default. My understanding was that we still wanted to only report errors in production by default but allow a mechanism to bypass this behavior (looks like I missed this in the last code review). Do we want to default to reporting all errors now?